wp-content/themes/wp-starter/plop-templates/block.json.hbs

```diff
@@ -0,0 +1,41 @@
+{
+  "name": "{{dashCase name}}",
+  "title": "{{titleCase name}}",
+  "description": "Displays the {{titleCase name}}",
+  "icon": "{{icon}}",
+  "category": "components",
+  "textdomain": "{{dashCase themeslug}}",
+  "keywords": ["{{dashCase name}}"],
+  "attributes": {
+    "align": {
+      "type": "string",
+      "default": "full"
+    },
+    "style": {
+      "type": "object",
+      "default": {
+        "spacing": {
+          "padding": {
+            "top": "var:preset|spacing|20",
+            "bottom": "var:preset|spacing|20",
+            "left": "var:preset|spacing|20",
+            "right": "var:preset|spacing|20"
+         }
+        }
+      }
+    }
+  },
+  "supports": {
+    "jsx": true,
+    "mode": false,
+    "alignWide": true,
+    "color": {
+      "background": true
+    }
+  },
+  -- STYLES HERE --
+  -- VARIATIONS HERE --
```

Comment:
We will either update these with real json for styles/variations or we will remove them on build.